### PR TITLE
[feature] Added last modified directive

### PR DESCRIPTION
**Pull request by alex-deaconu**
Directive moved from CH

Removed modal-lookup-textbox code
- Code wasn't reviewed and shouldn't have been merged/committed

Updated build, added lint and pretty tasks
Removed unneeded dependencies
Removed config folder; using standalone config

Updated readme

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@olegrise if you don't mind, please review. Thanks!

File: dist/js/last-modified.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+(function () {
+  "use strict";
+
+  angular.module("risevision.common.components.last-modified", [])
+    .directive("lastModified", ["$templateCache",
```
Comment (olegrise) [inline dist/js/last-modified.js]: @alex-deaconu do you think we should add `rv` prefix to our directives?

**Comment by olegrise** [comment]
:+1: Nicely done!

File: dist/js/last-modified.js (hunk 1 above)
Comment (alex-deaconu) [inline dist/js/last-modified.js]: @olegrise I don't disagree with the prefix, however we haven't done that anywhere else for existing AngularJS names. Do you think I should start doing it here? Thanks.

**Comment by alex-deaconu** [comment]
Merging PR, will address prefix issue separately if needed. Thanks.

File: dist/js/last-modified.js (hunk 1 above)
Comment (olegrise) [inline dist/js/last-modified.js]: @alex-deaconu not sure. Just wanted to know your opinion. Thanks.

**Pull request merged**